src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -60,12 +60,12 @@ const populateUISchemaForDisplay = (uiSchema, ionField) => {
   uiSchema.type = display.inputType;
   if (display.inputType === 'radio') {
     uiSchema.options = display.options;
+  } else if (display.inputType === 'select' && display.format === 'country-code') {
```

Comment:
nit: I think the format wont be present for all attributes, wondering if we should make sure it is not empty, perhaps something like this:
`_.isNotEmpty(display.format)`

Reply:
updated. Good call